README.md

```diff
@@ -12,24 +12,7 @@ Initial implementation created by [Kev](https://github.com/kevaundray), [Carlos]
 Redesigned by the [rust zkp](https://github.com/rust-zkp) team to have a backend which is compatible with the [arkworks](https://github.com/arkworks-rs) suite. This allows us to leverage the multitude of curves
```

Comment:
Should probably change this to ZK-Garage team and also change the link to https://github.com/ZK-Garage

Reply:
I fixed the README in my PR.

Reply:
True!! Let me change this!